File: spring-boot-oauth/src/main/java/sample/spring/boot/IndexController.java
package sample.spring.boot;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

/**
 * Created by yamashiro-r on 2017/02/12.
 */
@Controller
@RequestMapping("/")
public class IndexController {

    @GetMapping
    String index() {
        return "/index";
    }

}

File: spring-boot-kotlin/src/main/kotlin/sample/spring/boot/controller/IndexController.kt
package sample.spring.boot.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

/**
 * Created by yamashiro-r on 2017/02/25.
 */
@Controller
@RequestMapping("/")
class IndexController {

    @GetMapping
    fun index() = "index"

}